firestore-bigquery-export/firestore-bigquery-change-tracker/src/bigquery/index.ts

```diff
@@ -100,22 +100,71 @@ export class FirestoreBigQueryEventHistoryTracker
     return data;
   }
 
+  /**
+   * Check whether a failed operation is retryable or not.
+   * Reasons for retrying:
+   * 1) We added a new column to our schema. Sometimes BQ is not ready to stream insertion records immediately
+   *    with the new column.
+   */
+  private async isRetryableInsertionError(e) {
+    let isRetryable = true;
+    const expectedErrors = [
+      {
+        "message": "no such field.",
+        "location": "document_id"
+      }
+    ]
+    if (
+      e.response &&
+      e.response.insertErrors &&
+      e.response.insertErrors.errors
+    ) {
+      const errors = e.response.insertErrors.errors;
+      errors.map((error) => {
```

Comment:
nit: here and line 125, `forEach` would be more appropriate than `map` https://stackoverflow.com/a/34426513/4816918